pkg/model/uimodel.go

```diff
@@ -75,12 +89,18 @@ func (u *UIModel) View() string {
 		stats.PodsByPhase[v1.PodPending], stats.PodsByPhase[v1.PodRunning], stats.BoundPodCount)
 
 	fmt.Fprintln(&b)
-	for _, n := range stats.Nodes {
+	// dynamically calculate the number of lines we can fit per page
+	// taking into account header and footer text
+	u.paginator.PerPage = u.height - 6
```

Comment:
Looks good except that the node heights can vary.  Use this option and they'll be three lines per node:

```--resources=cpu,memory,pods`

Reply:
ah ok, I updated it to go completely dynamic.